.bcr/presubmit.yml

```
@@ -1,7 +1,9 @@
-bazel: 6.4.0
+matrix:
+  bazel: ["6.x"]
```

Want to be representative of our CI?
```suggestion
matrix:
  bazel: ["6.x", "7.x"]
```

---

We dont support 7.x yet though

---

🤦